Altis_Life.Altis/core/pmenu/fn_s_onCheckedChange.sqf

```diff
@@ -6,30 +6,30 @@
     Description:
     Switching it up and making it prettier..
 */
-private ["_option","_state"];
-_option = _this select 0;
-_state = _this select 1;
+params [
+    ["_option","",[""]],
+    ["_state",0,[0]]
+];
+if (_option isEqualTo "") exitWith {};
 
-switch (_option) do {
+switch _option do {
     case "tags": {
         if (_state isEqualTo 1) then {
-            life_settings_tagson = true;
             profileNamespace setVariable ["life_settings_tagson",true];
+            LIFE_ID_PlayerTags = addMissionEventHandler ["EachFrame", life_fnc_playerTags];
```

Comment:
Double check LIFE_ID_PlayerTags for these before adding/removing (sanity)?

Ditto below